package.json

```diff
@@ -3,11 +3,11 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "start": "npx nx run pxweb2:serve",
+    "start": "npm run build-style-dictionary && nx run pxweb2:serve",
```

Comment:
There is also a `prestart` script if you want to make it more tidy https://docs.npmjs.com/cli/v10/using-npm/scripts